PROVENANCE_FORMAT.md

```diff
@@ -1,19 +1,39 @@
-Each of the projects utilize the same base GitHub workflow SLSA provenance. The common fields of the SLSA provenance predicate attested to are below.
+# Provenance Format
 
-## Since v1.0.0
+The provenance format for each workflow conforms to the [SLSA v0.2 provenance
+format](https://slsa.dev/provenance/v0.2). Each of the projects populate the
+provenance with the same base information based on GitHub Actions. The common
+fields of the SLSA provenance predicate attested to are below.
+
+## Common Fields.
+
+### Builder
+
+`Builder.Id`: The builder ID refers to the fully qualified URI of the trusted builder's workflow and its reference.
 
-`BuildType`: This is the URI for the particular provenance builder, for example, the go or generic builder. 
 ```json
-  "buildType": "https://github.com/slsa-framework/slsa-github-generator/go@v1"
+"builder": {
+  "id": "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@refs/tags/v0.0.1"
+}
 ```
-`Builder.Id`: The builder ID refers to the fully qualified URI of the trusted builder's workflow and its reference. 
+
+### Build Type
+
+`buildType`: This is the URI for the particular provenance builder. For example, this is the `buildType` for the Go builder:
+
 ```json
- "builder": {
-      "id": "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@refs/tags/v0.0.1"
-  }
+"buildType": "https://github.com/slsa-framework/slsa-github-generator/go@v1"
 ```
-`Invocation`: Identifies the event that kicked off the build. This describes the workflow run and includes GitHub workflow event information, entrypoint, and parameters from trigger events. 
-`Invocation.configSource`: This describes the calling workflow's source and the entrypoint of the build.
+
+In SLSA the [buildType](https://slsa.dev/provenance/v0.2#buildType) defines how
+many other fields are formatted including all of those below.
+
+### Invocation
+
+`invocation`: Identifies the event that kicked off the build. This describes the workflow run and includes GitHub workflow event information, entrypoint, and parameters from trigger events.
+
+`invocation.configSource`: This describes the calling workflow's source and the entrypoint of the build.
```

Comment:
Would it make sense to tie this to the example by including the explicit field names in the description?
```suggestion
`invocation.configSource`: This describes the calling workflow's source (`uri` and `digest`) and the `entryPoint` of the build.
```